### DISCUSSION TITLE
Charging power disappeared from my InfluxDB backed Grafana dashboard

### DISCUSSION THREAD

**Discussion by joba-1** [pypolestar/polestar_api#205] [Q&A]
Took me a while to figure this out...
I have the InfluxDB integration activated in my homeassistant instance so I have easy access (e.g. by Grafana) to all data.
Was it intentional to move the charging power values (and possibly others that I don't use) of integration influx from measurement "state" to "W"?
Must have happened somewhere between June and today...

**Comment by jschlyter** [comment]
Look at https://github.com/pypolestar/polestar_api/blame/main/custom_components/polestar_api/sensor.py#L208 I cannot see this has changed in quite some time. Perhaps @leeyuentuen knows?

**Reply by leeyuentuen** [reply]
Not sure if i have done on this plugin. But maybe it was for the native support. I think for the miles and km i've done something with native value. So you can change the value in HA e.g. to meter, km, miles,...
